extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/OkHttpVaultClient.java

```diff
@@ -312,4 +385,17 @@ private URL getUrl(String path) {
         }
     }
 
+    private Map<String, String> getHealthParams(boolean isStandByOk, boolean isPerfStandByOk) {
+        Map<String, String> queryParams = new HashMap<>();
+        if (isStandByOk) {
+            queryParams.put("standbyok", "true");
+        }
+
+        if (isPerfStandByOk) {
+            queryParams.put("perfstandbyok", "true");
+        }
```

Comment:
```suggestion
        queryParams.put("standbyok", String.valueOf(isStandByOk));
        queryParams.put("perfstandbyok", String.valueOf(isPerfStandByOk));
```

Reply:
I have not done in this way to not create objects + parse things when it is false. Also notice that this map is iterated in another piece of code, so a readiness check is called several times + usually user will leave these with defaults (`false`) I thought that it will be a minor improvement (that's true) but at least an improvement.

Reply:
It was a minor nitpick. We can leave the code as is.